rotkehlchen/constants/assets.py

```diff
@@ -2327,3 +2327,15 @@
     protocol=None,
 )
 CONSTANT_ASSETS.append(A_RSPT)
+A_FOX = EthereumToken.initialize(
```

Comment:
You forgot how we do this here. We never edit this file directly. You need to edit the template in the same file and then run `python tools/scripts/generate_constant_assets.py`